### ISSUE TITLE
Disable error messaging

### ISSUE THREAD

**Issue by Drangonfly30** [TeamNewPipe/NewPipe#9986]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my problem isn't listed.
- [X] I'm aware that this is a request for NewPipe itself and that requests for adding a new service need to be made at [NewPipeExtractor](https://github.com/TeamNewPipe/NewPipeExtractor/issues).
- [X] I have taken the time to fill in all the required details. I understand that the feature request will be dismissed otherwise.
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Feature description

Whenever newpipe crashes, it gives an error message. I know it's a necessary thing for developers but sometimes it gets annoying having an error notification. A toggle to disable it will be good. I know sponserblock version has this thing but I think it should be native.

### Why do you want this feature?

It'll improve user experience.

**Comment by Cragsand** [comment]
For the recent weeks NewPipe shows an error message for every single video launched, even though it keeps playing normally. It gets a bit tedious since it obscures the bottom controls so it would be really nice to be able to turn error messages off. Anyway, love the app otherwise! Best regards

**Comment by glm4610** [comment]
@Cragsand Yes, I believe what you're describing is issue https://github.com/TeamNewPipe/NewPipe/issues/10808

**Comment by PLEB666** [comment]
Can this be implemented ? I hate seeing the error message at the bottom of newpipe everytime I do something. It pops up and i have to wait before I am able to interract with the currently playing video.
